lib/puppet.rb

```diff
@@ -36,6 +37,10 @@ module Puppet
   require 'puppet/environments'
 
   class << self
+    include FastGettext::Translation
+    GettextSetup.initialize(File.absolute_path('../locales', File.dirname(__FILE__)))
+    FastGettext.locale = GettextSetup.negotiate_locale(ENV["HTTP_ACCEPT_LANGUAGE"])
```

Comment:
Under what conditions do we expect `ENV["HTTP_ACCEPT_LANGUAGE"]` to be set? I would expect that this would be set when servicing an HTTP request, but this seems funny to always set even outside of a web service request.